src/sentry/utils/metrics.py

```diff
@@ -89,6 +89,8 @@ def incr(key, amount=1, instance=None, tags=None, skip_internal=True):
         internal.incr(key, instance, tags, amount)
     try:
         backend.incr(key, instance, tags, amount, sample_rate)
+        if not skip_internal:
+            backend.incr('internal_metrics.incr', key, tags, amount, sample_rate)
```

Comment:
`amount` here should always just be `1` since you're not incrementing the value from the key itself, we want to know how many times this key was used.

Do we even need to send along any `tags` or can we just pass `None` though explicitly? I'm not sure if tags here would yield anymore value.

If anything here, we should join the `key` and `instance` like we do internally, to get a better read on the real key names. You basically want this: https://github.com/getsentry/sentry/blob/master/src/sentry/utils/metrics.py#L59-L62

Reply:
Alternatively, you can leave off joining `key` and `instance` and just get the higher level usage of the key prefix. That might be more useful on second thought.